package.json

```diff
@@ -33,7 +33,6 @@
   },
   "homepage": "https://github.com/asyncapi/diff#readme",
   "dependencies": {
-    "@asyncapi/parser": "^1.5.1",
```

Comment:
You should use the parser in the tests, so please move this dep to the `devDependencies`.

Reply:
Cool, then I will fix the test in `diff.spec.ts` file